internal/rpc/user_device_data_api.go

```diff
@@ -85,6 +87,26 @@ func (s *userDeviceData) GetUserDeviceData(ctx context.Context, req *pb.UserDevi
 	}, nil
 }
 
+func (s *userDeviceData) GetRawDeviceData(ctx context.Context, req *pb.RawDeviceDataRequest) (*pb.RawDeviceDataResponse, error) {
+	//Create gRPC connection
+	conn, err := grpc.Dial("address:port", grpc.WithInsecure())
```

Comment:
instead of `"address:port"` you'll need to use the value from the settings (in `config` package).